src/snowflake/snowpark/session.py

```diff
@@ -265,6 +265,9 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+    def __repr__(self):
+        return f"<{self.__class__.__module__}.{self.__class__.__name__}: role={repr(self._conn._conn.role)}, database={repr(self._conn._conn.database)}, schema={repr(self._conn._conn.schema)}, warehouse={repr(self._conn._conn.warehouse)}>"
```

Comment:
Should we also add an account?

Reply:
Makes sense to me, I will add account in front of role.